.env.sample

```diff
@@ -1,4 +1,57 @@
+DriverData=C:\Windows\System32\Drivers\DriverData
+HOMEDRIVE=C:
+SESSIONNAME=Console
+TERMINAL_EMULATOR=JetBrains-JediTerm
+LOGONSERVER=\\DESKTOP-MUO3PGP
+SystemRoot=C:\Windows
+HOMEPATH=\Users\potic
+ChocolateyInstall=C:\ProgramData\chocolatey
+_OLD_VIRTUAL_PROMPT=$P$G
+LOCALAPPDATA=C:\Users\potic\AppData\Local
+APPDATA=C:\Users\potic\AppData\Roaming
+PROCESSOR_IDENTIFIER=AMD64 Family 23 Model 96 Stepping 1, AuthenticAMD
+PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW
+PSModulePath=C:\Program Files\WindowsPowerShell\Modules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules
+ProgramFiles(x86)=C:\Program Files (x86)
+PROMPT=(venv) $P$G
+OS=Windows_NT
+PROCESSOR_ARCHITECTURE=AMD64
+NUMBER_OF_PROCESSORS=6
+ComSpec=C:\Windows\system32\cmd.exe
+PROCESSOR_LEVEL=23
+_OLD_VIRTUAL_PATH=C:\Python311\Scripts\;C:\Python311\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\Git\cmd;C:\Program Files\Docker\Docker\resources\bin;C:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Users\potic\AppData\Local\Programs\Python\Python310\Scripts\;C:\Users\potic\AppData\Local\Programs\Python\Python310\;C:\Users\potic\AppData\Local\Microsoft\WindowsApps;;C:\Program Files\JetBrains\PyCharm Community Edition 2022.2\bin;;C:\Users\potic\AppData\Local\Programs\Microsoft VS Code\bin;C:\Program Files\JetBrains\PyCharm 2023.1\bin;;C:\Users\potic\AppData\Roaming\npm
+windir=C:\Windows
+USERDOMAIN_ROAMINGPROFILE=DESKTOP-MUO3PGP
+PyCharm Community Edition=C:\Program Files\JetBrains\PyCharm Community Edition 2022.2\bin;
+ProgramFiles=C:\Program Files
+TMP=C:\Users\potic\AppData\Local\Temp
+TEMP=C:\Users\potic\AppData\Local\Temp
+CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files
+OneDrive=C:\Users\potic\OneDrive
+OneDriveConsumer=C:\Users\potic\OneDrive
+USERDOMAIN=DESKTOP-MUO3PGP
+SystemDrive=C:
+COMPUTERNAME=DESKTOP-MUO3PGP
+ProgramData=C:\ProgramData
+ChocolateyLastPathUpdate=133407316658463712
+VIRTUAL_ENV=C:\Users\potic\PycharmProjects\py-cinema-full-stack\venv
+FPS_BROWSER_USER_PROFILE_STRING=Default
+CommonProgramFiles=C:\Program Files\Common Files
+__INTELLIJ_COMMAND_HISTFILE__=C:\Users\potic\AppData\Local\JetBrains\PyCharmCE2022.2\terminal\history\py-cinema-full-stack-history
```

Comment:
Variables till to 52 lines are not needed

Reply:
Still not fixed